### DISCUSSION TITLE
CPU cooler fans no longer being detected or controlled

### DISCUSSION THREAD

**Discussion by ninja-gaijin** [Rem0o/FanControl.Releases#1896] [Q&A]
I used to have these fans controlled by Fan Control but after not paying close attention and a number of updates, they seem to have disappeared. I now only have case fans and 'motherboard' fan showing. 

Were CPU fans removed at some stage?

**Comment by Rem0o** [comment]
Probably a rename that was done in the backend library (LibreHardwareMonitor).  You should have the same fans as before.
